controller/redirect_work_item_link_type_combination.go

```diff
@@ -0,0 +1,41 @@
+package controller
+
+import (
+	"strings"
+
+	"github.com/almighty/almighty-core/app"
+	"github.com/almighty/almighty-core/space"
+	"github.com/goadesign/goa"
+)
+
+// RedirectWorkItemLinkTypeCombinationController implements the redirect_work_item_link_type_combination resource.
+type RedirectWorkItemLinkTypeCombinationController struct {
+	*goa.Controller
+}
+
+// NewRedirectWorkItemLinkTypeCombinationController creates a redirect_work_item_link_type_combination controller.
+func NewRedirectWorkItemLinkTypeCombinationController(service *goa.Service) *RedirectWorkItemLinkTypeCombinationController {
+	return &RedirectWorkItemLinkTypeCombinationController{Controller: service.NewController("RedirectWorkItemLinkTypeCombinationController")}
+}
+
+// Create runs the create action.
+func (c *RedirectWorkItemLinkTypeCombinationController) Create(ctx *app.CreateRedirectWorkItemLinkTypeCombinationContext) error {
+	ctx.ResponseData.Header().Set("Location", redirectWorkItemLinkTypeCombinationsURL(ctx.RequestURI))
+	return ctx.MovedPermanently()
+}
+
+// // List runs the list action.
+// func (c *RedirectWorkItemLinkTypeCombinationController) List(ctx *app.ListRedirectWorkItemLinkTypeCombinationContext) error {
+// 	ctx.ResponseData.Header().Set("Location", redirectWorkItemLinkTypeCombinationsURL(ctx.RequestURI))
+// 	return ctx.MovedPermanently()
+// }
+
+// Show runs the show action.
+func (c *RedirectWorkItemLinkTypeCombinationController) Show(ctx *app.ShowRedirectWorkItemLinkTypeCombinationContext) error {
```

Comment:
this var name is quite long :-/, not a blocker but if you feel up to reduce its size